app/javascript/article-form/components/EditorActions.jsx

```diff
@@ -46,13 +46,14 @@ export const EditorActions = ({
   const schedule = publishedAtObj > now;
   const wasScheduled = passedData.publishedAtWas > now;
 
-  // if the article was saved as scheduled, and the user clears publishedAt in the post options, the save button text is changed to "Publish"
-  // to make it clear that the article is going to be published right away
-
   let saveButtonText;
-  if (schedule) {
+  if (isVersion1) {
+    saveButtonText = 'Save changes';
```

Comment:
The first and the last branches of if-else can be joined, but this way the conditions are easier to understand in my opinion